pkgs/test_core/lib/src/runner/configuration/load.dart

```diff
@@ -234,6 +235,18 @@ class _ConfigurationLoader {
       _error('Unknown reporter "$reporter".', 'reporter');
     }
 
+    var fileReporters = _getMap('file_reporter', key: (keyNode) {
```

Comment:
As currently written, this technically supports multiple file reporter mappings. Should I rename this to `file_reporters`?

Reply:
Good call. Yeah I think `file_reporters` makes sense in the yaml.